src/tcop/tcop.cpp

```diff
@@ -307,7 +307,7 @@ std::shared_ptr<Statement> TrafficCop::PrepareStatement(
 
 #ifdef LOG_DEBUG_ENABLED
     if (statement->GetPlanTree().get() != nullptr) {
-      LOG_DEBUG("Statement Prepared: %s", statement->GetInfo().c_str());
+      LOG_TRACE("Statement Prepared: %s", statement->GetInfo().c_str());
```

Comment:
Compression can be called only when the TileGroup is full. So it requires atleast 1001 tuples. LOG_DEBUG generates a large log file.